specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/preview/2020-01-01-preview/connectedClusters.json

```diff
@@ -630,8 +630,7 @@
           "type": "string",
           "description": "The mode of client authentication.",
           "enum": [
-            "Token",
-            "ClientCertificate"
+            "Token"
```

Comment:
removing value from enum will be breaking change which requires a new SDK version. While the API version is 2020-01-01-preview. It should be ok, let me double confirm on it.